src/app/landing/PartnerPicture.jsx

```diff
@@ -1,12 +1,38 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './PartnerPicture.css';
 
-const PartnerPicture = ({ src, alt, link }) => (
+const PartnerPicture = ({ imagePartialPath, alt = '', link }) => (
   <div className="PartnerPicture">
     <a href={link}>
-      <img src={src} alt={alt} />
+      <picture>
+        <source
+          className="OnboardingImage"
+          media="(min-width: 1224px)"
+          srcSet={`/img/partners/${imagePartialPath}_desktop.webp`}
+        />
+        <source
+          className="OnboardingImage"
+          media="(min-width: 1224px)"
+          srcSet={`/img/partners/${imagePartialPath}_desktop.jpg`}
+        />
+        <img
```

Comment:
No webp for mobile?

Reply:
Added!

Reply:
Wait, but also, don't you have to specify `type` for webp images? Otherwise I'm not sure this will work on Safari.
Also, in other places I see you had the mobile jpg both as a `source` and `img`. Here you don't need it?

Reply:
(Incidentally, this is why it generally helps to have a single component take care of all instances of logic like this. It avoids the risk of forgetting a version in 1 place, or having slightly different logic.)